modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -66,13 +66,13 @@ officially supported for making `/var` or a subdirectory of `/var` a separate
 partition, but not both. If you create more than one partition, Kubernetes
 will not be able to monitor them both.
 
-* Retain existing partitions: For a brownfield installation, where you are reinstalling on an existing {product-title} node and want to retain data partitions from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions.
+* Retain existing partitions: For a brownfield installation, where you are reinstalling on an existing {product-title} node and want to retain data partitions from your previous operating system, there are both boot arguments and options to `coreos-installer` that allow you to retain existing data partitions on a disk.
 
 [id="installation-user-infra-machines-advanced_vardisk_{context}"]
 === Creating a separate `/var` partition
 In general, disk partitioning for {product-title} should be left to the
-installer. However, there are cases where you might want to create a
-separate partition in a part of the filesystem that you expect to grow.
+installer. However, there are cases where you might want to create one or more
+separate partitions in a part of the disk filesystem that you expect to grow.
```

Comment:
Can it be more than one partition?

Reply:
The following paragraph says no.  RHCOS is fine with it, but I believe there's another OpenShift component that can only monitor one data filesystem.

Reply:
I think you are referring to this: https://access.redhat.com/articles/4766521

Reply:
Thanks, I've changed to refer to only a single partition for caution.